rep-2005.rst

```diff
@@ -0,0 +1,237 @@
+
+We aim that all of these repositories are also:
+
+* actively maintained and following accepted software development processes, including code review and testing.
+
+
+Change process
+==============
+
+This content is maintained at GitHub in the [REP repo](https://github.com/ros-infrastructure/rep).
+To propose changes, such as adding or removing items, make a pull request to that repo.
+The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
+
+
+Content
+=======
+
+* Buildsystem, package index and tooling around
+
+  * `ament/ament_cmake <https://github.com/ament/ament_cmake>`_
+  * `ament/ament_index <https://github.com/ament/ament_index>`_
+  * `ament/ament_package <https://github.com/ament/ament_package>`_
+  * `ament/ament_lint <https://github.com/ament/ament_lint>`_
+  * `ros/ros_environment <https://github.com/ros/ros_environment>`_
+  * `ros-infrastructure/bloom <https://github.com/ros-infrastructure/bloom>`_
+  * `ros-infrastructure/catkin_pkg <https://github.com/ros-infrastructure/catkin_pkg>`_
+  * `ros-infrastructure/rosdep <https://github.com/ros-infrastructure/rosdep>`_
+  * `ros-infrastructure/rosdistro <https://github.com/ros-infrastructure/rosdistro>`_
+  * `ros-infrastructure/ros_buildfarm <https://github.com/ros-infrastructure/ros_buildfarm>`_
+  * `ros2/ament_cmake_ros <https://github.com/ros2/ament_cmake_ros>`_
+
+* Third party packages
+
+  * `eProsima/foonathan_memory_vendor <https://github.com/eProsima/foonathan_memory_vendor>`_
+  * `ament/googletest <https://github.com/ament/googletest>`_
+  * `ament/uncrustify_vendor <https://github.com/ament/uncrustify_vendor>`_
+  * `ros2/console_bridge_vendor <https://github.com/ros2/console_bridge_vendor>`_
+  * `ros2/poco_vendor <https://github.com/ros2/poco_vendor>`_
+  * `ros2/spdlog_vendor <https://github.com/ros2/spdlog_vendor>`_
+  * `ros2/tinydir_vendor <https://github.com/ros2/tinydir_vendor>`_
+  * `ros2/tinyxml_vendor <https://github.com/ros2/tinyxml_vendor>`_
+  * `ros2/tinyxml2_vendor <https://github.com/ros2/tinyxml2_vendor>`_
+  * `ros2/yaml_cpp_vendor <https://github.com/ros2/yaml_cpp_vendor>`_
+
+* Utility functionality
+
+  * `micro-ROS/ros_tracing/ros2_tracing <https://gitlab.com/micro-ROS/ros_tracing/ros2_tracing>`_
+  * `osrf/osrf_pycommon <https://github.com/osrf/osrf_pycommon>`_
+  * `osrf/osrf_testing_tools_cpp <https://github.com/osrf/osrf_testing_tools_cpp>`_
+  * `ros/class_loader <https://github.com/ros/class_loader>`_
+  * `ros/pluginlib <https://github.com/ros/pluginlib>`_
+  * `ros2/eigen3_cmake_module <https://github.com/ros2/eigen3_cmake_module>`_
+  * `ros2/python_cmake_module <https://github.com/ros2/python_cmake_module>`_
+  * `ros2/rcutils <https://github.com/ros2/rcutils>`_
+  * `ros2/rcpputils <https://github.com/ros2/rcpputils>`_
+  * `ros2/ros_testing <https://github.com/ros2/ros_testing>`_
```

Comment:
I think that https://gitlab.com/ApexAI/performance_test has enough traction that it could be added here.

Reply:
Sounds good, added in 84bcee11fee8d59e7d35b2f263cd91ff93211dba.

Reply:
@ahcorde Can you comment on which additional repos might be useful here - specifically on the buildfarm integration?

Reply:
@ahcorde Friendly ping.

Reply:
For the performance test buildfarm integration we are using
 - https://github.com/ros2/buildfarm_perf_tests
 - https://github.com/ros2/performance_test

This last one it's a fork of https://gitlab.com/ApexAI/performance_test we should fix some issues because the master branch is not working with cycloneDDS. We have two PRs open in this repo. I will put some time into fix them.

Potentially this repo (https://github.com/ros-tooling/system_metrics_collector) from the ros tooling group could be interesting

Reply:
https://gitlab.com/ApexAI/performance_test and https://github.com/ros-tooling/system_metrics_collector are included.